tests/main.tf

```diff
@@ -0,0 +1,127 @@
+provider "aws" {
+  region = "us-east-1"
+}
+
+module "lx-instance-centos6" {
+  source = "git::https://github.com/plus3it/terraform-aws-watchmaker//modules/lx-instance/"
+
+  Name             = "lx-instance-centos6-${local.test_suffix}"
```

Comment:
I think it would be cleaner to use a prefix, instead of a suffix. That way all the resources sort together by name.

Reply:
agreed, updated